### PR TITLE
Add Flux to the integrations list

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>
Co-authored-by: Stefan Prodan <stefan.prodan@gmail.com>

/cc @stefanprodan

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 250 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~10 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by developer-guy** [comment]
Hello @tracymiranda, as the error says[^1], it seem I could not add Flux for both the "signed-with" and "integrations" sections:

```shell
1:18:15 PM: FATAL: Duplicate item: Cloud Native Computing Foundation (CNCF) Flux at path Sigstore / Integrations
1:18:15 PM: FATAL: Duplicate item: Cloud Native Computing Foundation (CNCF) Flux at path Sigstore / Signed With
```

[^1]: https://app.netlify.com/sites/ossflandscape/deploys/636638136eb8400008f72bb1#L180

**Comment by tracymiranda** [comment]
It won't allow duplicate names so have to change name to something like Flux (Integration).

File: landscape.yml [hunk 1]
```diff
@@ -771,6 +771,15 @@ landscape:
...
               how_to_verify_url: https://github.com/tektoncd/chains/releases
+          - item:
+            name: Flux (Integration)
+            homepage_url: https://fluxcd.io/
+            repo_url: https://github.com/fluxcd/flux
+            logo: flux.svg
+            twitter: https://twitter.com/fluxcd
+            crunchbase: https://www.crunchbase.com/organization/cloud-native-computing-foundation
+            extra:
+              how_to_verify_url: https://fluxcd.io/blog/2022/10/prove-the-authenticity-of-oci-artifacts/
```
Comment (stefanprodan) [inline landscape.yml:782]:
```suggestion
              how_to_verify_url: https://fluxcd.io/flux/cheatsheets/oci-artifacts/#signing-and-verification
```

**Review by stefanprodan — CHANGES_REQUESTED** [review]
Please let’s use a link to the official docs which are kept up to date.

**Comment by developer-guy** [comment]
Hi @tracymiranda, it seems that it is not only related to duplicating the names, but it also does not allow any duplication of the fields in the given spec🤔

**Comment by tracymiranda** [comment]
I think there is also an issue because that section is full (and it won't automatically resize stuff). Let me look at making some layout modifications first then we can address how to resolve this...

**Comment by tracymiranda** [comment]
OK I've made more space with #202 

@developer-guy looking at the kyverno entry, it is only the github url that cannot be duplicated, so either find a different url to link to or leave it out in one of the entries.

**Comment by tracymiranda** [comment]
Looks like it is complaining about the space issue, you may need to merge in my changes from #202

**Comment by stefanprodan** [comment]
Please let's swap labels, remove `Integration` from here and add `Signed With` to the other one, to match Kyverno. Anyway feels very strange to have this note in the project name.

File: landscape.yml [hunk 2]
```diff
@@ -771,6 +771,15 @@ landscape:
             crunchbase: https://www.crunchbase.com/organization/continuous-delivery-foundation-cdf
             extra:
               how_to_verify_url: https://github.com/tektoncd/chains/releases
+          - item:
+            name: Flux
+            homepage_url: https://fluxcd.io/
+            repo_url: https://github.com/fluxcd/flux
```
Comment (stefanprodan) [inline landscape.yml:777]:
```suggestion
            repo_url: https://github.com/fluxcd/flux2
```

**Review by stefanprodan — CHANGES_REQUESTED** [review]
The flux repo URL is wrong

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @developer-guy and @tracymiranda 🏅

**Comment by tracymiranda** [comment]
Awesome thank you!

**Pull request merged**